packages/core/src/model.d.ts

```diff
@@ -1054,7 +1054,7 @@ export interface FindOrCreateOptions<TAttributes = any, TCreationAttributes = TA
   /**
    * Default values to use if building a new instance
    */
-  defaults?: TCreationAttributes;
+  defaults?: Partial<TCreationAttributes>;
```

Comment:
Creation attributes are already partial, but only those that can be omitted as declared in the model.